test/components/canBeCreatedWithItems.ts

```diff
@@ -0,0 +1,55 @@
+import { IPokemon } from "../../src/components/Battles";
+import { INewPokemon } from "../../src/components/constants/Pokemon";
+import { FullScreenPokemon } from "../../src/FullScreenPokemon";
+import { it } from "../main";
+import { stubBlankGame } from "../utils/fakes";
+
+const pokemonTitle: string[] = "CHARMANDER".split("");
+
+it("Pokemon can be given an item when created", (): void => {
+    // Arrange
+    const fsp: FullScreenPokemon = stubBlankGame();
```

Comment:
There should be a blank line before each `// Act` comment (also the `// Assert`s)